VologramsToolkit/Scripts/VolPluginInterface.cs

```diff
@@ -136,19 +136,19 @@ private static void DebugCallbackFunction(int logType, string debugString)
         {
             case 0: 
                 if (interfaceLoggingLevel.HasFlag(VolEnums.LoggingLevel.Info))

```

Comment:
Could this switch be replaced with a single statement like the below (will need to add map.. methods)? 
if (geomLoggingLevel.HasFlag(mapLogTypeToLoggingLevel(logType)))
                    Debug.Log($"VOL_GEOM [mapLogTypeToDebugLogType(logType)] {debugString}");

Reply:
Hi Valeria, there are 2 checks - one is the type of message received and the other is if the user wants that type of message printed to Unity. There are probably nicer ways to write this but since it is a debug only function, I'm happy the way it is. 😃